### PR TITLE
Add some improvements in the modules documentation

### PR DESCRIPTION
**Pull request by iuriaranda**
- Use `terraform-docs` to generate the documentation of all modules
- Fix some issues with the md linter
- Document how to specify the teleport tokens in all modules

### CODE REVIEW

**Review by Komorebi-E — APPROVED** [review]
Looks good. The descriptions read well and are clear.

File: README.md [hunk 1]
```diff
@@ -83,52 +84,60 @@ The server will use Letsencrypt to retrieve a valid certificate for the Teleport
...
+| key_name | SSH key name for the EC2 instance. | string | - | yes |
+| letsencrypt_email | Email to use to register to letsencrypt. | string | `letsencrypt@skyscrapers.eu` | no |
+| project | A project where this setup belongs to. Only for naming reasons. | string | - | yes |
+| r53_zone | The Route53 zone where to add the Teleport DNS record. | string | - | yes |
+| root_vl_delete | Whether the root volume of the EC2 instance should be destroyed on instance termination. | string | `true` | no |
+| root_vl_size | Volume size for the root volume of the EC2 instance, in gigabytes. | string | `16` | no |
+| root_vl_type | Volume type for the root volume of the EC2 instance. Can be `standard`, `gp2`, or `io1`. | string | `gp2` | no |
+| subnet_id | Subnet id where the EC2 instance will be deployed. | string | - | yes |
+| teleport_auth_tokens | List of static tokens to configure in the Teleport server. *Note* that these tokens will be added "as-is" in the Teleport configuration, so they must be pre-fixed with the token type. See the official [documentation on static tokens](https://gravitational.com/teleport/docs/admin-guide/#static-tokens) for more info. | string | `<list>` | no |
```
Comment (duboisph) [inline README.md]: Can you add an example?

**Review by venturel — APPROVED** [review]
Apart from Philip's comment looks good

File: README.md (hunk 1 above)
Comment (iuriaranda) [reply README.md]: done

**Pull request merged**